spec/rubocop/cop/style/ternary_operator_spec.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+describe RuboCop::Cop::Style::TernaryOperator do
+  subject(:cop) { described_class.new }
+
+  it 'registers offense when the if branch and the else branch are ' \
+     'on a separate line from the condition' do
+    inspect_source(<<-RUBY.strip_indent)
+      a = cond ?
+        b : c
+    RUBY
+    expect(cop.offenses.size).to eq(1)
+  end
+
+  it 'registers an offense when the false branch is on a separate line' do
+    inspect_source(<<-RUBY.strip_indent)
+      a = cond ? b :
+          c
+    RUBY
+
+    expect(cop.offenses.size).to eq(1)
+  end
+
+  it 'registers an offense when everything is on a separate line' do
+    inspect_source(<<-RUBY.strip_indent)
+      a = cond ?
+          b :
+          c
+    RUBY
+
+    expect(cop.offenses.size).to eq(1)
+  end
+
+  it 'does not accept a single line ternary operator expression' do
+    inspect_source(<<-RUBY.strip_indent)
+      a = cond ? b : c
```

Comment:
The parsed AST should be the same regardless of the whitespace, and since the cop uses the AST only, these assertions don't really test any edge cases. 🙂 

Please also use the new `#expect_offense` matcher, e.g.:

```ruby
it 'registers an offense for ternary operators' do
  expect_offense(<<-RUBY.strip_indent)
    a = cond ? b : c
       ^^^^^^^^^^^^ Do not use ternary operators, use if / else instead.
  RUBY
end
```